BridgeEmulator/HueEmulator3.py

```diff
@@ -1519,9 +1523,17 @@ def do_GET(self):
                         self._set_end_headers(bytes(json.dumps({"name":"Group 0","lights": [l for l in bridge_config["lights"]],"sensors": [s for s in bridge_config["sensors"]],"type":"LightGroup","state":{"all_on":all_on,"any_on":any_on},"recycle":False,"action":{"on":False,"alert":"none"}},separators=(',', ':'),ensure_ascii=False), "utf8"))
                     elif url_pices[3] == "info" and url_pices[4] == "timezones":
                         self._set_end_headers(bytes(json.dumps(bridge_config["capabilities"][url_pices[4]]["values"],separators=(',', ':'),ensure_ascii=False), "utf8"))
+                    elif "scenes" == url_pices[3]: #trim lightstates for scenes
+                        scenelist = {}
+                        scenelist["scenes"] = copy.deepcopy(bridge_config["scenes"])
+                        for scene in list(scenelist["scenes"]):
+                            if ("type" in scenelist["scenes"][scene]) and ("GroupScene" == scenelist["scenes"][scene]["type"]):
+                                scenelist["scenes"][scene]["lights"] = {}
+                                scenelist["scenes"][scene]["lights"] = bridge_config["groups"][bridge_config["scenes"][scene]["group"]]["lights"]
+                        self._set_end_headers(bytes(json.dumps(scenelist["scenes"][url_pices[4]],separators=(',', ':'),ensure_ascii=False), "utf8"))
                     else:
                         self._set_end_headers(bytes(json.dumps(bridge_config[url_pices[3]][url_pices[4]],separators=(',', ':'),ensure_ascii=False), "utf8"))
-            elif len(url_pices) == 3 or url_pices[3] == "config": #used by applications to discover the bridge
+            elif (url_pices[2] == "nouser" or url_pices[2] == "none" or url_pices[2] == "config"): #used by applications to discover the bridge
```

Comment:
``` curl 192.168.10.113/api/test123
[{"error":{"type":1,"address":"/","description":"unauthorized user"}}]                                                                                                                                        ✔

 curl 192.168.10.113/api/test123/config
{"name":"Original Bridge","datastoreversion":"88","swversion":"1935074050","apiversion":"1.35.0","mac":"00:17:88:xxxxxxx","bridgeid":"001788FFFxxxxxx","factorynew":false,"replacesbridgeid":null,"modelid":"BSB002","starterkitid":""}                                                                                                                                                                                    ✔

  curl 192.168.10.113/api/
[{"error":{"type":4,"address":"/","description":"method, GET, not available for resource, /"}}]
```

so if there is no valid user but url_pices[2] == "config" we need to display the minimal config.

`(url_pices[2] == "nouser" or url_pices[2] == "none"` are useless there

Reply:
I didn't write this code. I just undid the patch that you wrote. This is original code.